test_allianceutils/tests/test_decorators.py

```diff
@@ -26,6 +26,8 @@ def my_method(self):
         obj.my_method.clear_cache()
         self.assertEqual(obj.my_method(), 3)
 
+        self.assertEqual(obj.my_method.__name__, 'my_method')
```

Comment:
Suggestion: since correct wrapping is being tested, I'd also add a random attribute to `my_method` and check that it's preserved.

Reply:
unfortunately it's not though - update_wrapper just assigns `'__module__', '__name__', '__qualname__', '__doc__',
                       '__annotations__'`